grakn-engine/src/main/java/ai/grakn/engine/backgroundtasks/distributed/ClusterManager.java

```diff
@@ -200,8 +202,12 @@ private void startZookeeperConnection() throws Exception{
      * Close all connections to Zookeeper
      */
     private void stopZookeeperConnection(){
-        cache.close();
-        failover.close();
+        if (cache!=null) {
```

Comment:
They are null when this instance of engine (`ClusterManager`) does not have leadership and has not started the scheduler